geometry_msgs/Pose[] camera_pose
string[] camera_id
time last_measurement_timestamp
